config.in

```diff
@@ -202,6 +202,7 @@ bindsym $mod+r mode "resize"
 # Read `man 5 sway-bar` for more information about this section.
 bar {
     position top
+    status_command while date +'%Y-%m-%d %l:%M:%S %p'; do sleep 1; done
```

Comment:
Perhaps with a comment explaining what this esoteric-looking command does?

Reply:
`# When the status_command prints a new line to stdout, swaybar updates. The default just shows the current date & time`